tools/deb/BUILD

```diff
@@ -11,7 +11,7 @@ pkg_tar(
 
 pkg_tar(
     name = "istio-systemd",
-    files = ["istio.service"],
+    files = ["istio-node-agent.service"],
```

Comment:
Should we call it `istio-auth-agent`, which sounds more on par with the pilot agent?

Reply:
Makes sense. Done

Reply:
Can you /lgtm it? Thanks.